sdk/schemaregistry/Azure.Data.SchemaRegistry/README.md

```diff
@@ -78,8 +74,7 @@ The following shows examples of what is available through the SchemaRegistryClie
 Register a schema to be stored in the Azure Schema Registry.
 
 ```C# Snippet:SchemaRegistryRegisterSchema
-string schemaName = "<schema_name>";
-string groupName = "<schema_group_name>";
+string schemaName = "employeeSample";
```

Comment:
Did you mean to remove the generic blurb here?

Reply:
I saw that other samples from other SDKs just make generic names for things that are irrelevant in actually running the sample code. Here, the name of the schema doesn't particularly matter.

Reply:
Kept this in the sample.